pyproject.toml

```
@@ -91,7 +91,11 @@ branch = true
 source = ["towncrier"]
 
 [tool.coverage.paths]
-source = ["src", ".nox/*/site-packages"]
+source = [
+    "src",
+    "*.nox/*/site-packages",
+    "*.nox\\*\\*\\site-packages"
+]
```

I guess this config with slashes each direction is compensating for now for the Windows issues discussed in https://github.com/nedbat/coveragepy/issues/991.  I'd suggest just using the 6.6.0b1 that has a fix for that but...  https://github.com/pypa/pip/issues/11565  I'm not sure, towncrier may or may not trigger that bug.  So maybe best to just wait and leave a comment here as reference.

```suggestion
source = [
    "src",
    "*.nox/*/site-packages",
    # required until coverage 6.6.0 is used: https://github.com/nedbat/coveragepy/issues/991
    "*.nox\\*\\*\\site-packages"
]
```